content/200-orm/200-prisma-client/500-deployment/301-edge/485-deploy-to-vercel.mdx

```diff
@@ -0,0 +1,458 @@
+}
+
+datasource db {
+  provider     = "mysql"
+  url          = env("DATABASE_URL")
+  relationMode = "prisma" // recommended for PlanetScale DBs
+}
+```
+
+Next, you need to set the `DATABASE_URL` environment variable in your `.env` file that's used both by Prisma and Next.js to read your env vars:
+
+```bash file=.env
+DATABASE_URL="mysql://32qxa2r7hfl3102wrccj:password@us-east.connect.psdb.cloud/demo-cf-worker-ps?sslaccept=strict"
+```
+
+#### 2. Install dependencies
+
+Next, install the required packages:
+
+```terminal
+npm install @prisma/adapter-planetscale
+npm install @planetscale/database
+```
+
+#### 3. Configure `postinstall` hook
+
+Next, add a new key to the `scripts` section in your `package.json`:
+
+```js file=package.json highlight=5;add
+{
+  // ...
+  "scripts": {
+    // ...
+    "postinstall": "prisma generate"
+  }
+}
+```
+
+#### 4. Migrate your database schema (if applicable)
+
+If you ran `npx prisma init` above, you need to migrate your database schema to create the `User` table that's defined in your Prisma schema (if you already have all the tables you need in your database, you can skip this step):
+
+```terminal
+npx prisma db push
+```
+
+#### 5. Use Prisma Client in an Edge Function to send a query to the database
+
+If you created the project from scratch, you can create a new edge function as follows.
+
+First, create a new API route, e.g. by using these commands:
+
+```terminal
+mkdir src/app/api
+mkdir src/app/api/edge
+touch src/app/api/edge/route.ts
+```
+
+Here is a sample code snippet that you can code to instantiate `PrismaClient` and send a query to your database in the new `app/api/edge/route.ts` file you just created:
+
+```ts file=app/api/edge/route.ts
+import { NextResponse } from 'next/server'
+import { PrismaClient } from '@prisma/client'
+import { PrismaPlanetScale } from '@prisma/adapter-planetscale'
+import { Client } from '@planetscale/database'
+
+export const runtime = 'edge'
+
+export async function GET(request) {
+  const client = new Client({ url: process.env.DATABASE_URL })
+  const adapter = new PrismaPlanetScale(client)
+  const prisma = new PrismaClient({ adapter })
+
+  const users = await prisma.user.findMany()
+
+  return NextResponse.json(users, { status: 200 })
+}
+```
+
+#### 6. Run the Edge Function locally
+
+Run the app with the following command:
+
+```terminal
+npm run dev
+```
+
+You can now access the Edge Function via this URL: [`http://localhost:3000/api/edge`](http://localhost:3000/api/edge).
+
+#### 7. Set the `DATABASE_URL` environment variable and deploy the Edge Function
+
+Run the following command to deploy your project with Vercel:
+
+```terminal
+npx vercel deploy
+```
+
+Note that once the project was created on Vercel, you will need to set the `DATABASE_URL` envrionment variable (and if this was your first deploy, it likely failed). You can do this either via the Vercel UI or by running the following command:
+
+```
+npx vercel env add DATABASE_URL
+```
+
```

Comment:
Anything to do after running this command or will it magically just start working?
How do I access the newly created (and now hopefully working) edge function?